boot/zephyr/sample.yaml

```diff
@@ -19,7 +19,8 @@ tests:
   sample.bootloader.mcuboot.usb_cdc_acm_recovery:
     tags: bootloader_mcuboot
     platform_allow:  nrf52840dongle/nrf52840
-    extra_args: DTC_OVERLAY_FILE="./usb_cdc_acm.overlay;app.overlay"
+    extra_args: EXTRA_CONF_FILE=./usb_cdc_acm_recovery.conf
```

Comment:
Can this be replaced with - list, like the integration_platform below?

Reply:
> Can this be replaced with - list, like the integration_platform below?

OK. Updated with the - list.